packages/fast-components-react-msft/src/carousel/carousel.tsx

```diff
@@ -110,15 +128,13 @@ class Carousel extends Foundation<
      */
     protected generateClassNames(): string {
         let className: string = get(this.props, "managedClasses.carousel", "");
-        let theme: string = "";
 
         if (this.getSlideTheme()) {
-            theme =
-                this.getSlideTheme() === SlideTheme.light
-                    ? get(this.props, "managedClasses.carousel__themeLight", "")
-                    : get(this.props, "managedClasses.carousel__themeDark", "");
+            className += this.assignSlideThemeClassName();
+        }
 
-            className += ` ${theme}`;
+        if (this.slideTransitionDirection) {
```

Comment:
See previous note, I would like this to check for null explicitly, empty string is a weird value to purposely assign